tests/src/test/java/alluxio/cli/fs/command/LoadCommandIntegrationTest.java

```diff
@@ -59,5 +60,8 @@ public void loadFile() throws IOException, AlluxioException {
     mFsShell.run("load", "/testFile");
     status = mFileSystem.getStatus(uri);
     assertTrue(status.getInAlluxioPercentage() == 100);
+    mFsShell.run("load", "--local", "/testFile");
```

Comment:
We should also test that "load --local" works when the file isn't already loaded.

Reply:
Indeed